cliapi/cliapi.py

```diff
@@ -0,0 +1,287 @@
+#!/usr/bin/python3
+
+# Guide for cliapi developers found here:
+# https://github.com/edlane/cliapi/blob/master/README.md
+
+import os
+
+if 'PYCHARM_DEBUG_ME' in os.environ:
```

Comment:
Just remembered that @rjschwei previously called out this code for being to "dev tool specific".
This will be removed.